src/ast_walk_interpreter.rs

```diff
@@ -316,12 +316,7 @@ pub fn call_func(func: &Function, arg_vals: &[Value]) -> Result<Option<Value>, R
             Ok(None)
         }
         Function::NativeReturning(_, ref native_fn) => Ok(Some(native_fn(arg_vals.to_vec())?)),
-        Function::User {
-            ref param_names,
-            ref body,
-            ref env,
-            ..
-        } => {
+        Function::User { ref param_names, ref body, ref env, .. } => {
```

Comment:
Ah, not wanting these rustfmt changes will require you to rebase on master (or at least copy rustfmt.toml) ensure that it's updated, and then run rustfmt.